mechanics/micromechanics/include/micromechanics/agent_generic_storage.h

```diff
@@ -0,0 +1,68 @@
+#pragma once
+
+#include <common/base_agent_generic_storage.h>
+#include <common/types.h>
+
+#include "agent_data.h"
+#include "cell_interface.h"
+
```

Comment:
add pragma to disable windows warnings at the beginning and end of file here same as is in `reactions-diffusion/biofvm/include/biofvm/agent_generic_storage.h`